test/game_team_test.rb

```diff
@@ -13,8 +13,10 @@ def setup
                                 :head_coach=>"John Tortorella",
                                 :goals=>2,
                                 :shots=>8,
-                                :tackles=>44
+                                :tackles=>44,
```

Comment:
Did you mean to add this comma here?

Reply:
Nice catch @leahriffell! Just corrected it.